components/retro-go/rg_gui.c

```diff
@@ -1266,6 +1269,15 @@ void rg_gui_debug_menu(const rg_gui_option_t *extra_options)
     snprintf(heap_free, 20, "%d+%d", stats.freeMemoryInt, stats.freeMemoryExt);
     snprintf(block_free, 20, "%d+%d", stats.freeBlockInt, stats.freeBlockExt);
     snprintf(uptime, 20, "%ds", (int)(rg_system_timer() / 1000000));
+    float batteryPct, batteryVlt;
+    if (rg_input_read_battery(&batteryPct, &batteryVlt))
+    {
+        snprintf(battery_info, sizeof(battery_info), "%.2f%% | %.2fV", batteryPct, batteryVlt);
+    }
+    else
+    {
+        snprintf(battery_info, sizeof(battery_info), "Battery: N/A");
```

Comment:
This will end up displaying Battery: Battery: N/A, no? I think the string should just be "N/A".